486_Project/Fight_Class.py

```diff
@@ -207,19 +206,19 @@ def warrior_first_vs_bad_warrior(self):
                         self.fighting_player.warrior_hp += 10
                         if self.fighting_player.warrior_hp > self.fighting_player.warrior_max_hp:
                             self.fighting_player.warrior_hp = self.fighting_player.warrior_max_hp
```

Comment:
This block of code is used multiple times. It might be worth adding a "heal" method to the Player class that will perform this check.